packages/ui/src/forum/modals/CreateThreadModal/CreateThreadDetailsModal.tsx

```diff
@@ -37,30 +43,32 @@ export const CreateThreadDetailsModal = ({
 
   return (
     <>
-      <Modal onClose={hideModal} modalSize="l">
-        <ModalHeader title="Create a thread" onClick={hideModal} />
-        <ModalBody>
-          <RowGapBlock gap={24}>
-            <ForumBreadcrumbsList
-              categoryBreadcrumbs={breadcrumbs ?? []}
-              threadBreadcrumb={{ id: '-1', title: 'New Thread' }}
-              nonInteractive
-            />
-            <RowGapBlock gap={16}>
-              <TextMedium light>Please make sure your title will be clear for users</TextMedium>
-              <InputComponent label="Topic of the thread" id="field-topic">
-                <InputText id="field-topic" value={topic} onChange={(event) => setTopic(event.target.value)} />
-              </InputComponent>
-              <InputComponent label="Description" required inputSize="auto" id="field-description">
-                <CKEditor
-                  id="field-description"
-                  onChange={(_, editor) => setDescription(editor.getData())}
-                  onReady={(editor) => editor.setData(description ?? '')}
-                />
-              </InputComponent>
+      <ScrolledModal onClose={hideModal} modalSize="l">
+        <ModalHeader title="Create gh thread" onClick={hideModal} />
```

Comment:
Thank you for the PR, it's exactly what is needed. Please just fix the typo here:
```suggestion
        <ModalHeader title="Create a thread" onClick={hideModal} />
```